taskotron_python_versions/two_three.py

```diff
@@ -108,10 +97,11 @@ def check_two_three(path):
                     log.debug('Requires Python {}'.format(py_version))
                     py_versions[py_version] = name
 
-    return hdr[rpm.RPMTAG_NAME], py_versions
+    package.set_python_versions(py_versions)
```

Comment:
Do we actually need to keep the dictionary, or are we only interested in versions? If we store a set of the dict's keys, wouldn't that suffice?